openwisp_firmware_upgrader/hardware.py

```diff
@@ -39,6 +39,84 @@
                 'ar71xx-generic-ubnt-airrouter-squashfs-sysupgrade.bin',
                 {'label': 'Ubiquiti AirRouter', 'boards': ('Ubiquiti AirRouter',)},
             ),
+            (
+                'zbt-wg3526-16M-squashfs-sysupgrade.bin',
+                {'label': 'ZBT-WG3526 (16M)', 'boards': ('ZBT-WG3526 (16M)',)},
+            ),
+            (
+                'tl-wr2543-v1-squashfs-sysupgrade.bin',
+                {
+                    'label': 'TP-Link TL-WR2543N/ND',
+                    'boards': ('TP-Link TL-WR2543N/ND',),
+                },
+            ),
+            (
+                'cpe210-v3-squashfs-sysupgrade.bin',
+                {'label': 'TP-LINK CPE210 v3', 'boards': ('TP-LINK CPE210 v3',)},
+            ),
+            (
+                'cpe510-v3-squashfs-sysupgrade.bin',
+                {'label': 'TP-LINK CPE510 v3', 'boards': ('TP-LINK CPE510 v3',)},
+            ),
+            (
+                'archer-c7-v1-squashfs-sysupgrade.bin',
+                {'label': 'TP-Link Archer C7 v1', 'boards': ('tplink,archer-c7-v1',)},
+            ),
+            (
+                'archer-c7-v2-squashfs-sysupgrade.bin',
+                {
+                    'label': 'TP-Link Archer C7 v2',
+                    'boards': ('TP-Link Archer C7 v2', 'TP-Link Archer C7 v3'),
+                },
+            ),
+            (
+                'archer-c7-v4-squashfs-sysupgrade.bin',
+                {'label': 'TP-Link Archer C7 v4', 'boards': ('TP-Link Archer C7 v4',)},
+            ),
+            (
+                'archer-c7-v5-squashfs-sysupgrade.bin',
+                {'label': 'TP-Link Archer C7 v5', 'boards': ('TP-Link Archer C7 v5',)},
+            ),
+            (
+                'tl-wr902ac-v3-squashfs-sysupgrade.bin',
```

Comment:
seems to me that this one and the ones above are missing the target (eg: `ar71xx-generic`)

Reply:
fixed all of them with ath79 to be future proof.